supertokens_python/recipe/emailpassword/utils.py

```diff
@@ -185,12 +169,15 @@ def __init__(self,
                  form_fields_for_password_reset_form: List[NormalisedFormField],
                  form_fields_for_generate_token_form: List[NormalisedFormField],
                  get_reset_password_url: Callable[[User, Dict[str, Any]], Awaitable[str]],
-                 create_and_send_custom_email: Callable[[User, str, Dict[str, Any]], Awaitable[None]]):
+                 create_and_send_custom_email: Union[Callable[[User, str, Dict[str, Any]], Awaitable[None]], None] = None):
```

Comment:
You need to remove this from the normalised type. Same goes for email verification normalised type.

Reply:
@KShivendu missed this?